### ISSUE TITLE
is snu4 a snorna?

### ISSUE THREAD

**Issue by Antonialock** [pombase/curation#1718]
it's labelled as an snRNA but it localizes to the nucleolus and they call it a snorna in PMID:28765164

**Comment by ValWood** [comment]
I don't know...but this resource calls snu4 and snRNA.....
http://spliceosomedb.ucsc.edu/proteins/12122

I don't know if this is historical though...the same applies to our other snu*, they seem to be nucleolar...

**Comment by mah11** [comment]
It looks pretty well established by sequence similarity and mutant phenotypes as the pombe U4 snRNA, famous for its role in splicing. All three [U4/U6.U5] tri-snRNP snRNAs have been found in the nucleolus in Xenopus (e.g. PMID:12939253), but only transiently. Maybe something similar is going on in pombe. Anyway, because they're involved in splicing, which isn't particularly associated with the nucleolus, they're all still called snRNAs, not snoRNAs.

tl;dr answer: "no"
